stringutil/detemplate.go

```diff
@@ -12,7 +12,8 @@ import (
 // the rendering.
 // NOTE: This function does not support sequential template variables.
 // e.g. 'test {{id1}}{{id2}}' as there is no way of knowing where one would end
-//      and the next would begin.
+//
+//	and the next would begin.
```

Comment:
The formatter doesn't like when tabs don't match. Use this suggestion to prevent it from adding a newline

```suggestion
// and the next would begin.
```